libs/blocks/marketo/marketo.js

```diff
@@ -186,11 +188,73 @@ export const formSuccess = (formEl, formData) => {
   return false;
 };
 
+export const handleIframeTimeout = (el) => {
+  const searchParams = new URLSearchParams(window.location.search);
+  const config = getConfig();
+  const iframe = document.querySelector('iframe[src*="/index.php/form/XDFrame"]');
+  const formEl = el.querySelector('form');
+  let iframeTimeout = null;
+
+  const handleIframeReady = (event) => {
+    if (event.origin === 'https://engage.adobe.com') {
+      const message = JSON.parse(event.data);
+      if (message.mktoReady) {
+        if (iframeTimeout) clearTimeout(iframeTimeout);
+        const errorOverlay = el.querySelector('.marketo-overlay');
+        if (errorOverlay) {
+          errorOverlay.remove();
+          if (formEl) formEl.inert = false;
+        }
+        window.removeEventListener('message', handleIframeReady);
+      }
+    }
+  };
+
+  const decorateOverlay = async () => {
+    const cookieSize = document.cookie.length > 4096 ? '> 4k' : '< 4k';
+    window.lana?.log(`Marketo iframe timeout - Cookie Size ${cookieSize}`, { tags: 'error,marketo' });
```

Comment:
Instead of using error as a tag, you can change the severity
```suggestion
    window.lana?.log(`Marketo iframe timeout - Cookie Size ${cookieSize}`, { tags: 'marketo', severity: 'e' });
```
Docs: https://wiki.corp.adobe.com/display/WCMSOps/LANA+-+Log+Always+Never+Assume#LANALogAlwaysNeverAssume-Parameters